.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,32 @@
-Closes #
+<!--
 
-<!-- Thank you for contributing to Storybook! If your PR is related to an issue, provide the number(s) above; if it resolves multiple issues, be sure to break them up (e.g. "closes #1000, closes #1001"). -->
+Thank you for contributing to Storybook! Please submit all PRs to the `next` branch unless they are specific to the current release. Storybook maintainers cherry-pick bug and documentation fixes into the `main` branch as part of the release process, so you shouldn't need to worry about this. For additional guidance: https://storybook.js.org/docs/react/contribute/how-to-contribute
+
+-->
 
 ## What I did
 
 <!-- Briefly describe what your PR does -->
 
-## How to test
+<!-- If your PR is related to an issue, provide the number(s) below; if it resolves multiple issues, be sure to break them up (e.g. "closes #1000, closes #1001"). -->
+
+Closes #
```

Comment:
@kasperpeulen — Does this still need to be the first line to be picked up by release scripts?

Reply:
I don't think the release script does this anymore, but I will put this line back at the top just in case we reinstate it later.

Reply:
I don't think the release script does this anymore, but I will put this line back at the top just in case we reinstate it later.